demo/src/pages/channels/channelClient.tsx

```diff
@@ -0,0 +1,82 @@
+import { useChannelClient } from "openfin-react-hooks";
+import * as Prism from "prismjs";
+import React, { useEffect, useState } from "react";
+import styles from "./Channel.module.css";
+
+const CHANNEL_NAME = "demo";
+
+const codeExample = `import {useChannelClient} from "openfin-react-hooks";
+  const CHANNEL_NAME = "demo";
+
+  const Component = () => {
+      const { client } = useChannelClient(CHANNEL_NAME);
+
+      useEffect(() => {
+        // register for push messages from the ChannelProvider
+        if (client) {
+          client.register("pushMessage", (payload: any) => setPushMessage(payload));
```

Comment:
Could you remove the `: any` type from this example? I'm trying to keep them in JavaScript